pageStyles/constants.scss

```diff
@@ -1,4 +1,5 @@
-﻿
+﻿//Default Header Width
+$welcomeHeaderWidth: 760px;
```

Comment:
Is this only used in index.module.scss? It would be better to put it in that file if so. It gives more context for understanding the name, and it makes it quicker to see what value the variable has.